### PR TITLE
Update dependency "language-tags"

### PR DESCRIPTION
**Pull request by ThaFireDragonOfDeath**
## PR Type

Other (updated outdated dependency)

## PR Checklist

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made for the appropriate packages.
- [x] Format code with the latest stable rustfmt.
- [x] (Team) Label with affected crates and semver status.

## Overview

I searched with cargo-outdated for outdated dependency's and found that the dependency "language-tags" was updated from 0.2(.2) to 0.3(.1) some days ago.
So I bumped the dependency version in the Cargo.toml to "0.3", ran cargo test and fixed all failed tests (tests ran on Manjaro Linux with rust 1.51).
The new version of language-tags have some relevant breaking changes which broken 4 tests:

1. The macro "langtag!" doesn't exists anymore (use LanguageTag::parse instead).
2. The struct "LanguageTag" doesn't implement the "Default" trait anymore.
3. The fields of the struct "LanguageTag" are now private.

This is my first pull request so sorry if I forgot something or messed something up.

### CODE REVIEW

**Comment by JohnTitor** [comment]
Could you also update the changelog on actix-http? Otherwise LGTM.

**Comment by ThaFireDragonOfDeath** [comment]
> Could you also update the changelog on actix-http? Otherwise LGTM.

Done.

File: src/http/header/accept_language.rs [hunk 1]
```diff
@@ -24,14 +24,12 @@ crate::__define_common_header! {
     /// # Examples
     ///
     /// ```
-    /// use language_tags::langtag;
+    /// use language_tags::LanguageTag;
```
Comment (JohnTitor) [inline src/http/header/accept_language.rs]:
```suggestion
```

File: src/http/header/accept_language.rs [hunk 2]
```diff
@@ -24,14 +24,12 @@ crate::__define_common_header! {
...
     /// use actix_web::HttpResponse;
-    /// use actix_web::http::header::{AcceptLanguage, LanguageTag, qitem};
+    /// use actix_web::http::header::{AcceptLanguage, qitem};
```
Comment (JohnTitor) [inline src/http/header/accept_language.rs]:
```suggestion
    /// use actix_web::http::header::{AcceptLanguage, LanguageTag, qitem};
```

File: src/http/header/accept_language.rs [hunk 3]
```diff
@@ -40,16 +38,16 @@ crate::__define_common_header! {
     /// ```
     ///
     /// ```
-    /// use language_tags::langtag;
+    /// use language_tags::LanguageTag;
```
Comment (JohnTitor) [inline src/http/header/accept_language.rs]:
```suggestion
```

File: src/http/header/accept_language.rs [hunk 4]
```diff
@@ -40,16 +38,16 @@ crate::__define_common_header! {
...
     /// use actix_web::HttpResponse;
     /// use actix_web::http::header::{AcceptLanguage, QualityItem, q, qitem};
```
Comment (JohnTitor) [inline src/http/header/accept_language.rs]:
```suggestion
    /// use actix_web::http::header::{AcceptLanguage, LanguageTag, QualityItem, q, qitem};
```

File: src/http/header/content_language.rs [hunk 5]
```diff
@@ -24,28 +24,28 @@ crate::__define_common_header! {
     /// # Examples
     ///
     /// ```
-    /// use language_tags::langtag;
+    /// use language_tags::LanguageTag;
```
Comment (JohnTitor) [inline src/http/header/content_language.rs]:
```suggestion
```

File: src/http/header/content_language.rs [hunk 6]
```diff
@@ -24,28 +24,28 @@ crate::__define_common_header! {
...
     /// use actix_web::HttpResponse;
     /// use actix_web::http::header::{ContentLanguage, qitem};
```
Comment (JohnTitor) [inline src/http/header/content_language.rs]:
```suggestion
    /// use actix_web::http::header::{ContentLanguage, LanguageTag, qitem};
```

File: src/http/header/content_language.rs [hunk 7]
```diff
@@ -24,28 +24,28 @@ crate::__define_common_header! {
...
     ///     ContentLanguage(vec![
-    ///         qitem(langtag!(en)),
+    ///         qitem(LanguageTag::parse("en").unwrap()),
     ///     ])
     /// );
     /// ```
     ///
     /// ```
-    /// use language_tags::langtag;
+    /// use language_tags::LanguageTag;
```
Comment (JohnTitor) [inline src/http/header/content_language.rs]:
```suggestion
```

File: src/http/header/content_language.rs [hunk 8]
```diff
@@ -24,28 +24,28 @@ crate::__define_common_header! {
...
     /// use actix_web::HttpResponse;
     /// use actix_web::http::header::{ContentLanguage, qitem};
```
Comment (JohnTitor) [inline src/http/header/content_language.rs]:
```suggestion
    /// use actix_web::http::header::{ContentLanguage, LanguageTag, qitem};
```

**Review by JohnTitor — COMMENTED** [review]
Sorry, I missed that we've re-exported `language_tags::LanguageTag` already, let's use it to simplify.

**Comment by ThaFireDragonOfDeath** [comment]
> Sorry, I missed that we've re-exported `language_tags::LanguageTag` already, let's use it to simplify.

Done.

**Review by robjtede — APPROVED** [review]

**Pull request merged**